### PR TITLE
Event refactor

### PR DESCRIPTION
**Pull request by nathanielmit**
I've tested the changes and they seem to be working

### CODE REVIEW

File: server/schemas.js [hunk 1]
```diff
@@ -35,12 +35,14 @@ const Authentication = mongoose.model('Authentication', authenticationSchema);
...
 const eventSchema = new mongoose.Schema({
     eventID: { type: String, required: true, unique: true},
-    eventName: { type: String, required: true},
-    location: { type: String },
-    collaborators: [{ type: String }],
-    viewers: [{ type: String }],
+    title: { type: String, required: true},
+    location: [ { address: String } ],
+    collaborators: [{ username: String, name: String }],
+    viewers: [{ username: String, name: String }],
```
Comment (hokansom) [inline server/schemas.js]: Can we also get the photoUrl, even if it's an empty string for viewers and collaborators?

File: server/schemas.js [hunk 2]
```diff
@@ -35,12 +35,14 @@ const Authentication = mongoose.model('Authentication', authenticationSchema);
...
     date: { type: Date},
-    past: { type: Boolean }
+    past: { type: Boolean },
+    description: { type: String },
+    color: { type: String, default: '#FFFFFF' }
```
Comment (hokansom) [inline server/schemas.js]: If it isn't too big of a deal, could you please update the default to be "#8be4d2"? (It's the main app color)

**Comment by nathanielmit** [comment]
I made the changes Morgan requested (added photoURL and changed default color)

**Comment by nathanielmit** [comment]
I added the date modifications requested for this week and updated the database populating script and it looks like it's working from my testing.

File: server/schemas.js (hunk 1 above)
Comment (nathanielmit) [reply server/schemas.js]: done!

File: server/schemas.js (hunk 2 above)
Comment (nathanielmit) [reply server/schemas.js]: done!

**Review by Clotonervo — APPROVED** [review]

**Pull request merged**